api/Models/User.cs

```diff
@@ -0,0 +1,19 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace YonderfulApi.Models
+{
+    public class User
+    {
+        [Key]
+        public int Id { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+
+        /// role == 0 - user
```

Comment:
Define an enum for the role type, instead of using "magic" strings.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum

Reply:
Matei already defined an enum in his PR: https://github.com/nicuvlad7/yonderful/pull/18/files 
Use this one once his PR is merged.